$OpenBSD: patch-t_t4062-diff-pickaxe_sh,v 1.1 2016/12/04 15:29:46 dcoppa Exp $

RE_DUP_MAX is 255 on OpenBSD, while it's 32767 on Linux

--- t/t4062-diff-pickaxe.sh.orig	Sat Dec  3 20:10:24 2016
+++ t/t4062-diff-pickaxe.sh	Sat Dec  3 20:13:12 2016
@@ -15,7 +15,7 @@ test_expect_success setup '
 	git commit -m "A 4k file"
 '
 test_expect_success '-G matches' '
-	git diff --name-only -G "^0{4096}$" HEAD^ >out &&
+	git diff --name-only -G "^0{255}" HEAD^ >out &&
 	test 4096-zeroes.txt = "$(cat out)"
 '
 
